docs/veux_plugin.md

```diff
@@ -0,0 +1,42 @@
+---
+id: vuex_plugin
+title: Vuex Plugins
+sidebar_label: Vuex Plugins
+---
+
+When you're working with plugins is slightly tricky have access to the veu object on its context because you don't have direct access to it. The best way to get around this limitation is to dynamically register a store in a created hook on your root component. It is also necessary to namespace your store due to the module nature of Kiln plugins. For more info on `Vuex modules` see official [documentation](https://vuex.vuejs.org/guide/modules.html)
```

Comment:
```suggestion
When you're working with plugins is slightly tricky to get access to the vue object on its context because you don't have direct access to it. The best way to get around this limitation is to dynamically register a store in a created hook on your root component. It is also necessary to namespace your store due to the module nature of Kiln plugins. For more info on `Vuex modules` see official [documentation](https://vuex.vuejs.org/guide/modules.html)
```

Reply:
Done!!